app/assets/javascripts/actions/story.js

```diff
@@ -2,6 +2,9 @@ import actionTypes from './actionTypes';
 import { sendSuccessNotification, sendErrorNotification, sendDefaultErrorNotification } from './notifications';
 import { wait } from '../services/promises';
 import { storyScopes } from '../libs/beta/constants';
+import httpService from '../services/httpService';
+import projectStoriesService from '../services/stories';
+import changeCase from 'change-object-case';
```

Comment:
`httpService` and `changeCase` are not being used here. Is it supposed to be used somewhere in this file ?